libs/auth/src/dipped_smart_card_auth.ts

```diff
@@ -130,10 +130,22 @@ async function logAuthEventIfNecessary(
           newAuthStatus.wrongPinEnteredAt !==
             previousAuthStatus.wrongPinEnteredAt
         ) {
-          await logger.log(LogEventId.AuthPinEntry, newAuthStatus.user.role, {
-            disposition: LogDispositionStandardTypes.Failure,
-            message: 'User entered incorrect PIN.',
-          });
+          if (newAuthStatus.lockedOutUntil) {
+            await logger.log(
+              LogEventId.AuthPinEntryLockout,
+              newAuthStatus.user.role,
+              {
+                disposition: LogDispositionStandardTypes.Failure,
+                message: `User entered incorrect PIN. Maximum attempts exceeded, locked out until: ${newAuthStatus.lockedOutUntil.toString()}`,
+                lockedOutUntil: newAuthStatus.lockedOutUntil.toString(),
+              }
+            );
+          } else {
+            await logger.log(LogEventId.AuthPinEntry, newAuthStatus.user.role, {
+              disposition: LogDispositionStandardTypes.Failure,
+              message: 'User entered incorrect PIN.',
+            });
+          }
```

Comment:
> can you think of any other weird auth cases to make sure to log?

The one other one that comes to mind is hitting a session time limit. We do log that you got logged out in that case but don't indicate why.

We could probably accomplish this by changing the `reason` field here:

https://github.com/votingworks/vxsuite/blob/44e8bd8c4fae2b02b59ac41ab7e79174e96c331f/libs/auth/src/dipped_smart_card_auth.ts#L432-L436

And adding a reason field to this log:

https://github.com/votingworks/vxsuite/blob/44e8bd8c4fae2b02b59ac41ab7e79174e96c331f/libs/auth/src/dipped_smart_card_auth.ts#L169-L172

Like we have for the failed login log:

https://github.com/votingworks/vxsuite/blob/44e8bd8c4fae2b02b59ac41ab7e79174e96c331f/libs/auth/src/dipped_smart_card_auth.ts#L108-L110

Open to other approaches too of course!